modules/wri_featured_tweet/config/install/views.view.featured_tweet.yml

```diff
@@ -1,4 +1,3 @@
-# Do not delete the uuid for this view.
```

Comment:
Nit pick: Can you leave in this comment for me? To keep future me honest.

Reply:
of course! didn't mean to remove it